README.md

```diff
@@ -23,6 +23,7 @@ Website: [https://kymatio.github.io](https://kymatio.github.io)
 kymatio requires:
 
 * Python (>= 3.6)
+* Anaconda
```

Comment:
oh sorry I did not notice you had summoned him seconds before I did :)

Reply:
Do we actually require conda?

Reply:
come to think of it, we don't
it's possible to install pytorch binaries with pip only, although installing torch from source requires conda
perhaps we can just remove this line?

Reply:
@eugenium I was only quoting the PyTorch README:
> If you are installing from source, we highly recommend installing an Anaconda environment. You will get a high-quality BLAS library (MKL) and you get a controlled compiler version regardless of your Linux distro.

I think we can remove the line, even though 95% of our users will end up using an Anaconda or Miniconda distribution. This bullet point list at the top of the README is meant as a simple description of the most important dependencies, not as a full-fledged installation guide

Reply:
Yeah I think there was an issue with the MKL-based NumPy that is used by default in Conda. It would probably be enough to say that Conda is strongly recommended.

Best thing would obviously to figure out how to do it without Conda at all.

Reply:
Our package does not require anaconda.

If anything, pytorch installation is easier using anaconda, but that is not strictly our problem.

So I am also for removing this as a requirement and stating it as a recommendation (and maybe adding that this is for getting pytorch to work easily)

Reply:
Ok. Works for me.